hack/generator/pkg/astmodel/primitive_type.go

```diff
@@ -12,28 +12,29 @@ import (
 // PrimitiveType represents a Go primitive type
 type PrimitiveType struct {
 	name string
+	zero string
 }
 
 // IntType represents a Go integer type
-var IntType = &PrimitiveType{"int"}
+var IntType = &PrimitiveType{"int", "0"}
 
 // UInt64Type represents a Go uint64 type
-var UInt64Type = &PrimitiveType{"uint64"}
+var UInt64Type = &PrimitiveType{"uint64", "0"}
 
 // UInt32Type represents a Go uint32 type
-var UInt32Type = &PrimitiveType{"uint32"}
+var UInt32Type = &PrimitiveType{"uint32", "0"}
 
 // StringType represents the Go string type
-var StringType = &PrimitiveType{"string"}
+var StringType = &PrimitiveType{"string", "\"\""}
 
 // FloatType represents the Go float64 type
-var FloatType = &PrimitiveType{"float64"}
+var FloatType = &PrimitiveType{"float64", "0"}
 
 // BoolType represents the Go bool type
-var BoolType = &PrimitiveType{"bool"}
+var BoolType = &PrimitiveType{"bool", "false"}
 
 // AnyType represents the root Go interface type, permitting any object
-var AnyType = &PrimitiveType{"interface{}"}
+var AnyType = &PrimitiveType{"interface{}", "&struct{}{}"}
```

Comment:
This should be nil. (Although I guess it's academic since there shouldn't be any `AnyType`s left when this would happen?)

Reply:
Academic, but let's fix anyway.